dynamic/dynamic_message.go

```diff
@@ -1430,22 +1430,6 @@ func (m *Message) mergeFrom(pm proto.Message) error {
 		if prop.Tag == 0 {
 			continue // one-of or special field (such as XXX_unrecognized, etc.)
 		}
-		rv := src.FieldByName(prop.Name)
-		var keyProps, valProps *proto.Properties
-		if rv.Kind() == reflect.Map {
-			sfd, ok := src.Type().Elem().FieldByName(prop.Name)
-			if !ok {
-				panic(fmt.Sprintf("Could not get field %q for type %v", prop.Name, src.Type().Elem()))
-			}
-			keyProps = &proto.Properties{}
-			valProps = &proto.Properties{}
-			// NB: Stinks to have re-parse these every time. Would be nicer if proto.Properties exported
-			// its mkeyprop and mvalprop fields!
-			keyProps.Parse(sfd.Tag.Get("protobuf_key"))
-			valProps.Parse(sfd.Tag.Get("protobuf_val"))
```

Comment:
I can't remember why I had added this block. These things are completely unused, so I removed them.